tic_tac_toe/game.py

```diff
@@ -24,7 +28,18 @@ def _position_is_valid(position):
 
     Returns True if given position is valid, False otherwise.
     """
-    pass
+    valid_positions = [
+            (0,0), (0,1), (0,2),
+            (1,0), (1,1), (1,2),
+            (2,0), (2,1), (2,2),
+        ]
+    
+    for p in valid_positions:
```

Comment:
Thanks for the input. I agree with your suggestions. It also allows for extensibility.